src/Module/CreateCommand.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-tooling for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-tooling/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Tooling\Module;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Zend\Expressive\Tooling\Module\Exception;
+
+class CreateCommand extends Command
+{
+    use CommandCommonTrait;
+
+    const HELP = <<< 'EOT'
+Create a new middleware module for the application.
+
+- Creates an appropriate module structure containing a source code tree,
+  templates tree, and ConfigProvider class.
+- Adds a PSR-4 autoloader to composer.json, and regenerates the
+  autoloading rules.
+- Registers the ConfigProvider class for the module with the application
+  configuration.
+EOT;
+
+    const HELP_ARG_MODULE = 'The module to create and register with the application.';
+
+    /**
+     * Configure command.
+     */
+    protected function configure()
+    {
+        $this->setDescription('Create and register a middleware module with the application');
+        $this->setHelp(self::HELP);
+        $this->addDefaultOptionsAndArguments();
+    }
+
+    /**
+     * Execute command
+     *
+     * Executes command by creating new module tree, and then executing
+     * the "register" command with the same module name.
+     *
+     * {@inheritDoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $module = $input->getArgument('module');
+        $composer = $input->getOption('composer') ?: 'composer';
+        $modulesPath = $this->getModulesPath($input);
+
+        try {
+            $creation = new Create();
+            $message = $creation->process($module, $modulesPath, $this->projectDir);
+            $output->writeln(sprintf('<info>%s</info>', $message));
+        } catch (Exception\RuntimeException $e) {
+            $console = $this->getErrorConsole($output);
+            $console->writeln('<error>Error during execution:</error>', true, STDERR);
```

Comment:
Now we have different signature of `writeln` method - from symfony, so there are two params: `$messages` and `$options`.